Documentation/bootkube-upgrades.md

```diff
@@ -1,13 +1,13 @@
 
-# Upgrading Self-hosted Kubernetes
+# Upgrading self-hosted Kubernetes
 
 [Self-hosted](bootkube.md) Kubernetes clusters schedule Kubernetes components such as the apiserver, kubelet, scheduler, and controller-manager as pods like other applications (except with node selectors). This allows Kubernetes level operations to be performed to upgrade clusters in place, rather than by re-provisioning.
 
 Let's upgrade a self-hosted Kubernetes v1.4.1 cluster to v1.4.3 as an example.
 
 ## Inspect
 
-Show the control plane daemonsets and deployments which will need to be updated.
+Shows the control plane daemonsets and deployments which will need to be updated.
```

Comment:
Oh ! my impression was the tool "shows" the control pane daemonsets and deployments.